cache.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/DmitriyVTitov/size"
```

Comment:
We should avoid using external libraries.

Reply:
dependency removed